validator/src/main.rs

```diff
@@ -1543,6 +1543,12 @@ pub fn main() {
                 .default_value(default_rpc_max_request_body_size)
                 .help("The maximum request body size accepted by rpc service"),
         )
+        .arg(
+            Arg::with_name("rpc_prometheus_metrics")
+                .long("rpc-prometheus-metrics")
```

Comment:
```suggestion
            Arg::with_name("enable_rpc_prometheus_metrics")
                .long("enable-rpc-prometheus-metrics")
```
Not sure if there is already this inconsistency, but I see one arg that starts with `enable` below, it makes sense to me for boolean flags.

About `rpc` in the name, I’m not sure, on the one hand `--enable-rpc-prometheus-metrics` sounds like it would enable Prometheus metrics about RPC calls, which is not what the feature does. But on the other hand, it does listen on the RPC endpoint :thinking: I think just `--enable-prometheus-metrics` would be slightly clearer, but I don’t have a strong opinion on it.

Reply:
Yeah... if we don't want to tie it to the `rpc` name, it's better to omit this prefix.
IMO `--enable-prometheus-metrics` sounds better.

Anyway, one of the reasons I've added the `rpc` prefix is because the config parameter goes to the Rpc service config, and also because the endpoint is on the RPC port.

Reply:
changed to `enable-prometheus-metrics`